std_msgs/Header header
int64 prop_time             # prop_time reading
int64 prop_time_variance    # prop_time_variance reading
